### PR TITLE
Vis dg

### LINKED ISSUE #105: Add vis feature to DG

### PR DESCRIPTION
**Pull request by wwweert123**
fixes #105

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM!

**Review by NeoMinWei — APPROVED** [review]
LGTM
the Xchart acknowledgements could be put under the acknowledgements portion of the DG

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1,17 +1,34 @@
 # Developer Guide
 
 ## Acknowledgements
-- **round() method in Cashflow.java**
+
```
Comment (NeoMinWei) [inline docs/DeveloperGuide.md:4]: thanks for adding acknowledgements with similar formats

Comment (NeoMinWei) [inline docs/DeveloperGuide.md:4]: thanks for adding acknowledgements with similar formats

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**